packages/toolpad-app/src/toolpad/AppEditor/PageEditor/PageOptionsPanel.tsx

```diff
@@ -40,22 +40,40 @@ export default function PageOptionsPanel() {
       <Typography variant="subtitle1">Page:</Typography>
       <NodeNameEditor node={page} />
       <PageTitleEditor node={page} />
-      <TextField
-        select
-        defaultValue="shell"
-        value={page.attributes.display}
-        onChange={handleDisplayModeChange}
-        label="Display mode"
-        fullWidth
+      <Tooltip
+        arrow
+        placement="left-start"
+        title={
+          <Typography variant="inherit">
+            Control how the navigation panel is rendered in the final application. Read more in the{' '}
+            <Link
+              href="https://mui.com/toolpad/concepts/display-mode"
```

Comment:
This is a 301, slowing down the page load speed.

```suggestion
              href="https://mui.com/toolpad/concepts/display-mode/"
```

Fixed in ddffda5885f97c1ed7ac87dfe2a822c0e8672309